portal-api/src/v1/api.js

```diff
@@ -471,6 +471,36 @@ const getPortalFacilityAmendment = async (facilityId, amendmentId) => {
   }
 };
 
+/**
+ * Upserts a draft amendment for a portal facility in the database.
+ * @param {Object} params
+ * @param {string} params.dealId - id of the deal with the relevant facility.
+ * @param {string} params.facilityId - id of the facility to amend.
+ * @param {import('@ukef/dtfs2-common').PortalFacilityAmendmentUserValues} params.amendmentUpdate - the draft amendment to be upserted.
+ * @param {import('@ukef/dtfs2-common').AuditDetails} params.auditDetails - The audit details for the update.
+ * @returns {Promise<(import('@ukef/dtfs2-common').PortalFacilityAmendmentWithUkefId)>} - the amendment
+ */
+const putPortalFacilityAmendment = async ({ dealId, facilityId, amendmentUpdate, auditDetails }) => {
+  try {
+    const response = await axios({
+      method: 'put',
+      url: `${DTFS_CENTRAL_API_URL}/v1/portal/facilities/${facilityId}/amendments`,
+      headers: headers.central,
+      data: {
+        dealId,
+        facilityId,
+        amendment: amendmentUpdate,
```

Comment:
To keep it consistent can we rename the argument as `amendment`?

Reply:
I've renamed, thanks